packages/react-core/src/components/CodeBlock/CodeBlockAction.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+
+export interface CodeBlockActionProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the code block action */
+  children?: React.ReactNode;
+  /** Additional classes passed to the code block action */
+  className?: string;
+}
+
+export const CodeBlockAction: React.FunctionComponent<CodeBlockActionProps> = ({
+  children = null,
+  className,
+  ...props
+}: CodeBlockActionProps) => (
+  <div className={css('pf-c-code-block__actions-item', className)} {...props}>
```

Comment:
> Is .pf-c-code-block__actions-item a placeholder class? 

Yep. I suspect we will want that element and it will help us avoid a breaking change and be more consistent with the way we might use it by adding it now as opposed to later.